env0/resource_environment.go

```diff
@@ -348,6 +348,29 @@ func setEnvironmentSchema(d *schema.ResourceData, environment client.Environment
 		d.Set("approve_plan_automatically", !*environment.RequiresApproval)
 	}
 
+	if environment.LatestDeploymentLog.WorkflowFile != nil && len(environment.LatestDeploymentLog.WorkflowFile.Environments) > 0 {
```

Comment:
This is for setting sub environments ids.
(This data is only available in the response).